### DISCUSSION TITLE
submitting service requests via audio

### DISCUSSION THREAD

**Discussion by pwalsh** [govflow/govflow#14] [Ideas]
A major value proposition of the platform we are building is to be able to accept and process service requests from a number of "input channels" - basically, wherever the citizen is, we want to provide a handler for her to submit a request to her jurisdiction.

A number of platforms allow recording (and sending) audio. This is becoming increasingly popular on chat platforms like WhatsApp, Facebook Messenger, and Apple Messages.

We think it would be useful to be able to process audio as a service request. There are a bunch of things to take into consideration:

- raw transcription of audio to text via some existing APIs
- entity detection, and parsing of the raw text that is derived from the transcription
- confirmation with the user that what we extracted is what she intended?
- ability to make manual curation by admins of such a request (probably applies to all free text-type service requests)

cc @idoivri @amirozer

**Comment by idoivri** [comment]
Voice enabled tickets are very important for accessibility imho.
